### ISSUE TITLE
Enhanced New Event Page

### ISSUE THREAD

**Issue by abhay-dev2901** [anjalit03/Dazzle-and-Delight#17]
Hey @anjalit03 I want to enhance the UI of New event Page . Please assign this to me under hacktoberfest 2024.
<img width="1440" alt="Screenshot 2024-10-28 at 1 05 47 AM" src="https://github.com/user-attachments/assets/bf0281a2-79de-4307-82d2-358c4f8f402e">

**Comment by abhay-dev2901** [comment]
@anjalit03 Please consider PR #19  for the desired changes.

**Comment by abhay-dev2901** [comment]
@anjalit03 I am waiting for your response

**Comment by anjalit03** [comment]
yes I have checked the pr and mentioned a change please review that
